src/omnisharp/omnisharpManager.ts

```diff
@@ -11,16 +11,16 @@ import { PlatformInformation } from '../shared/platform';
 import { modernNetVersion } from './omnisharpPackageCreator';
 
 export class OmnisharpManager {
-    private readonly latestVersionFileServerPath = 'releases/versioninfo.txt';
-
     private readonly installPath = '.omnisharp';
+    private readonly githubRepoUrl = 'https://github.com/OmniSharp/omnisharp-roslyn';
+    private readonly githubApiUrl = 'https://api.github.com/repos/OmniSharp/omnisharp-roslyn';
 
     public constructor(
         private downloader: OmnisharpDownloader,
         private platformInfo: PlatformInformation,
         // Only the tests set this. Instead of making this configurable,
         // we should probably just mock the HTTP requests, not create an entire mock HTTP server.
-        private serverUrl: string = 'https://roslynomnisharp.blob.core.windows.net'
+        private serverUrl: string | undefined = undefined
```

Comment:
Continue to use this for tests to override. However, now that we need separate urls for downloading builds and finding latest version, we do not initialize this and allow each method to supply the correct production url.